config.json

```
@@ -1077,6 +1077,20 @@
           "control_flow_loops",
           "pointers"
         ]
+      },
+      {
+        "slug": "yacht",
+        "name": "Yacht",
+        "uuid": "dd360788-1dff-4a5d-89af-5bbd1199ae86",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2,
```

How about a 4? We have Eliud's eggs rated as a 2 and that's a fair bit less complicated.

```suggestion
        "difficulty": 4,
```

---

Whatever you find reasonable. Difficulties are all over the place. `spiral-matrix` is also a 4 and I would say that's more difficult than this. I updated it to 4 now.

---

Agreed, many of these need to be revisited